jdaviz/configs/imviz/plugins/coords_info/coords_info.vue

```diff
@@ -2,5 +2,6 @@
     <div v-if="pixel" style="white-space: nowrap;">
         <b v-if="pixel">Pixel </b>{{ pixel }}&nbsp;&nbsp;<b v-if="value">Value </b>{{ value }}<br>
         <b v-if="world">World </b>{{ world }}<br>
+        <b v-if="world">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</b>{{ world_deg }}<br>
```

Comment:
Maybe @kecnry knows a better way to do this... 😅 

Kinda hard to get them align perfectly from image to image, but hopefully this is good enough without having to specify a special `div` for each sub-element.

Reply:
You should be able to use here I think, to make everything line up without this workaround.

Reply:
I can think of many *other* ways to do this... but they all require nesting more elements or providing a fixed width for your empty `<b>` element.  But I think unless we want to structure this further, this is probably fine for now.

Reply:
So should I try out the `<v-col>` stuff, or leave it as `&nbsp;` like it is 1999?

Reply:
I guess using columns would be a little more font-safe (for non-fixed-width fonts where nbsp might not line up well) and would be more future proof to maintain down the road